ratis-docs/content/cli.md

```diff
@@ -0,0 +1,60 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Ratis-shell
+Ratis-shell is the command line interface of Ratis.
+The following command can be invoked in order to get the basic usage:
+
+```shell
+$ ratis sh
+Usage: ratis sh [generic options]
+         [election [transfer] [stepDown] [pause] [resume]]
+         [group [info] [list]]
+         [peer [add] [remove] [setPriority]]
+         [snapshot [create]]
+```
+Note: Ratis-shell is currently only experimental, and compatibility story is not considered for the time being
```

Comment:
@codings-dan , thanks for the update.  Let's move the Note up and highlight it.
```
# Ratis-shell

Ratis-shell is the command line interface of Ratis.

> **Note**:
> Ratis-shell is currently only **experimental**.
> The compatibility story is not considered for the time being.

The following command can be invoked in order to get the basic usage:

```shell
$ ratis sh
Usage: ratis sh [generic options]
         [election [transfer] [stepDown] [pause] [resume]]
         [group [info] [list]]
         [peer [add] [remove] [setPriority]]
         [snapshot [create]]
```

Reply:
done